tk_toolchain/cmd_line_tools/tk_build_qt_resources/__main__.py

```diff
@@ -0,0 +1,152 @@
+# Copyright (c) 2024 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+####################################################################################
+
+import argparse
+import re
+import subprocess
+
+
+def process_import_line(module, import_text):
+    return (
+        f"from {import_text} import {module}\n"
+        f"for name, cls in {module}.__dict__.items():\n"
+        f"    if isinstance(cls, type): globals()[name] = cls\n"
+    )
+
+
+def verify_compiler(compiler):
+    try:
+        print("Verifying compiler: ", compiler)
+        version_compiler = subprocess.check_output(
```

Comment:
Useless variable version_compiler, Just return it
```suggestion
        return subprocess.check_output(
```